src/ui/pages/GameUI.jsx

```diff
@@ -46,46 +66,47 @@ const GameUI = () => {
   }, [gameEngine, uiProxy]);
 
   return (
-    <div css={{ display: 'flex', flexDirection: 'column', gap: '10px', margin: '20px', height: '300px' }}>
-      <div css={{ display: 'flex', gap: '10px', alignItems: 'center' }}>
-        {running ? (
-          <button
-            css={btn}
-            type='button'
-            onClick={() => {
-              gameEngine.stop();
-              setRunning(false);
-            }}
-          >
-            Stop
-          </button>
-        ) : (
-          <button
-            css={btn}
-            type='button'
-            onClick={() => {
-              if (lose) {
-                reset();
-              }
-              gameEngine.start();
-              setRunning(true);
-            }}
-          >
-            Start
-          </button>
-        )}
-        <button css={btn} type='button' onClick={reset}>
-          Reset
+    <div css={wrapper}>
+      <PageHeader>Level 1</PageHeader>{' '}
+      {running ? (
+        <button
+          css={btn}
+          type='button'
+          onClick={() => {
+            gameEngine.stop();
+            setRunning(false);
+          }}
+        >
+          Stop
         </button>
-        <span>Distance: {distance}</span>
-      </div>
+      ) : (
+        <button
+          css={btn}
+          type='button'
+          onClick={() => {
+            if (lose) {
+              reset();
+            }
+            gameEngine.start();
+            setRunning(true);
+          }}
+        >
+          Start
+        </button>
+      )}
+      <button css={btn} type='button' onClick={reset}>
+        Reset
+      </button>
       {lose && (
         <span css={{ fontSize: '35px', textTransform: 'uppercase', position: 'absolute', left: '190px', top: '110px' }}>
           You Lose
         </span>
       )}
-      <canvas css={{ width: '512px', height: '512px' }} height={512} width={512} id='GameCanvas' ref={ref} />
+      <div css={gameBorder}>
+        <canvas css={{ width: '1150px', height: '512px' }} height={512} width={512} id='GameCanvas' ref={ref} />
```

Comment:
Tu jak aspect ratio wysokości i szerokości z CSSa nie będzie się zgadzać z atrybutami HTML to obraz będzie rozciągnięty. (na nasze potrzeby warto żeby to były wielokrotności 32, bo każdy poziom będzie miał takie rozmiary)

Reply:
własnie o co to chodzi z tym css który jest tu wpisany? Potrzebne jest to wszystko?